DESCRIPTION

```diff
@@ -159,7 +159,9 @@ Suggests:
     tgp,
     TH.data,
     wavelets,
-    xgboost (>= 0.6-2)
+    xgboost (>= 0.6-2),
+    sp,
```

Comment:
Is `sp` still needed for `gstat` We already have `sf`. Not sure if `gstat` has already migrated though.

Reply:
I think we can remove sp from the mlr DESCRIPTION file as gstat also works by passing a dataframe containing longitude and latitude columns (that must be specified). You can check this on the [gstat doc](https://www.rdocumentation.org/packages/gstat/versions/1.1-6/topics/gstat)